pallets/vault-registry/src/tests.rs

```diff
@@ -1297,7 +1297,7 @@ mod get_vaults_below_premium_collaterlization_tests {
 			let collateral1 = 49;
 
 			let id2 = vault_id(4);
-			let issue_tokens2: u128 = 50;
```

Comment:
did tests fail or where does this come from?

Reply:
@ebma  the test failed because of the order of the elements in the vec. 
To keep the asserts as is, i increased the value of `issue_tokens2`, to make sure that _vault_id 4_ always comes first before _vault_id 3._

Reply:
Okay got it